### PR TITLE
Media chrome 0.11.0 update

### PR DESCRIPTION
**Pull request by gkatsev**

### CODE REVIEW

**Review by luwes — APPROVED** [review]
👍

File: packages/mux-player/README.md [hunk 1]
```diff
@@ -225,22 +225,23 @@ Offers a way to turn off keyboard shortcuts that should not be enabled.
...
-| `f`   | Toggle fullscreen |
-| ⬅     | Seek back 10s     |
-| ➡     | Seek forward 10s  |
+| Key     | Behavior                     |
+| ------- | ---------------------------- |
+| `Space` | Toggle Playback              |
+| `k`     | Toggle Playback              |
+| `m`     | Toggle mute                  |
+| `f`     | Toggle fullscreen            |
+| `c`     | Toggle captions or subtitles |
```
Comment (cjpillsbury) [inline packages/mux-player/README.md:242]: It doesn't bother me *at all* that the orders of these are not consistent wrt docs... nope nope.

**Review by cjpillsbury — APPROVED** [review]
👌

File: packages/mux-player/README.md (hunk 1 above)
Comment (gkatsev) [reply packages/mux-player/README.md:242]: going to randomize the order of these across files

**Pull request merged**